client/src/App.jsx

```diff
@@ -53,11 +53,17 @@ export default function App() {
 
         // BUG: SOMETIMES THERE ARE DUPLICATES!!!!!!!!!
```

Comment:
if the bug is fixed, we can remove this comment. also would you be able to write another comment explaining why we are multiplying by 2 its confusing if someone other than us reads it. or maybe you reading it tomorrow cus you have goldfish memory.

Reply:
Yo plz goldfish have very good memories as per MY RESEARCH